README.md

```diff
@@ -16,15 +16,73 @@
 # limitations under the License.
 #
 -->
+
 # Apache APISIX Dashboard
 
-Dashboard for [Apache APISIX](https://github.com/apache/incubator-apisix-dashboard)
+Dashboard for [Apache APISIX](https://github.com/apache/apisix-dashboard)
 
-## Deploy with Docker (currently)
+## Deploy with Docker
 
 Please refer to [Deploy with Docker README](./compose/README.md)
 
-## More
+## Deploy Manually
+
+### Build the manager-api
+
+The `manager-api` is used to provide APIs for Dashboard, just like a bridge between the Apache APISIX and the Dashboard. Here are the steps to build it manually:
+
+1. We need `MySQL/Golang` to be preinstalled.
+
+```sh
+# e.g Initialization for MySQL
+$ mysql –uroot –p123456 
+> source $PROJECT_ROOT_FOLDER/api/script/db/schema.sql
+```
+
+2. Start the Apache APISIX.
+
+[Please follow this guide](https://github.com/apache/apisix#configure-and-installation)
+
+3. Set environment variables
+
+According to your local deployment environment, modify the environment variables in `$PROJECT_ROOT_FOLDER/api/run/run.sh`
+
+4. Build
+
+```sh
+$ go build -o manager-api $PROJECT_ROOT_FOLDER/api
+```
+
+5. Run
+
+```sh
+$ cd $PROJECT_ROOT_FOLDER
+$ chmod +x $PROJECT_ROOT_FOLDER/api/run/run.sh
```

Comment:
why need chmod?

Reply:
@gxthrj Please take a look at this comment, tks.

Reply:
done. remove the chmod, and give `run.sh` executable permissions.